plugins/modules/efs_tag.py

```diff
@@ -124,7 +125,7 @@ def main():
     '''
     argument_spec = dict(
         resource=dict(required=True),
-        tags=dict(type='dict', required=True),
+        tags=dict(type='dict', required=True, aliases=['resource_tags']),
         purge_tags=dict(type='bool', default=False),
```

Comment:
@tremble Should't we uniform the default behaviour of purge_tags by defaulting to true even here?

Reply:
I wasn't sure.  I wanted to get the "easy" bits in first, and was going to follow up with that later

Reply:
Adding the aliases was also something I can backport safely